web/server/README.md

```diff
@@ -96,6 +96,24 @@ openssl req -newkey rsa:2048 -nodes -sha512 -x509 -days 365 -keyout key.pem -out
 openssl speed rsa2048 rsa4096
 ```
 
+### Select TLS version
+
+Since version 1.21 Netdata accepts `tls key` and `tls certificate` as replacement of the options `ssl key` and `ssl certificate`, 
+ but you do not need to change your `netdata.conf` case you are using old `ssl` variables, because old options are kept due compatibility.
```

Comment:
```suggestion
Beginning with version 1.21, Netdata accepts `tls key` and `tls certificate` as replacement of the settings `ssl key` and `ssl certificate`. 
You do not need to change your `netdata.conf` if you are using the previous `ssl` settings, because these are kept for backwards compatibility.
```

Reply:
I used all suggestions that you gave me, thank you!